core/jazz_ui/Jenkinsfile_Platform

```diff
@@ -210,6 +211,11 @@ node ()  {
         echo "displaying env file : ${envFile}"
       }
     }
+    stage('Run build.website') {
```

Comment:
- Run the stage only when `build.website` file exists
- Rename the stage to `Execute Pre Build Hooks`